home/models.py

```diff
@@ -212,19 +224,25 @@ class Meta:
 
     def clean(self, value):
         result = super().clean(value)
+        num_vars_in_msg = result["message"].count("{{")
+        errors = {}
+        if num_vars_in_msg > 0:
+            num_example_values = len(result["example_values"])
+            if num_vars_in_msg != num_example_values:
+                errors["example_values"] = ValidationError(
+                    "The number of example values provided does not match the number of variables used in the template"
```

Comment:
Maybe we could add the number of example values provided, and the number of variables that we counted in the message, to this error message? eg. `The number of example values provided (2) does not match the number of variables used in the template (3)`

Reply:
Cool, done